nexus/reconfigurator/planning/src/example.rs

```diff
@@ -166,6 +251,18 @@ impl ExampleSystemBuilder {
         )
         .unwrap();
         builder.set_rng_seed((&self.test_name, "ExampleSystem make_zones"));
+
+        // Add as many external IPs as is necessary for external DNS zones. We
+        // pick a start address arbitrarily in the 10.x.x.x range.
```

Comment:
Tiny nit - should we use one of the reserved `TEST-NET-n` ranges (https://en.wikipedia.org/wiki/Reserved_IP_addresses)?

Reply:
Sure. Looks like service IPs use TEST-NET-1 so I picked TEST-NET-2 for these.